AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3261,6 +3261,7 @@ bmwfaq.org#?#.Notices[-ext-has='a[href="//www.bmwfaq.org/help/cookies"]']
 ! JS rules
 !
 cinemaxx.de#%#document.cookie = "showCookiesConsent=true; path=/;";
+warsteiner-gruppe.de#%#AG_onLoad(function() { if(document.querySelector(".gdpr-preface").isvisible = true) { Cookies.set('gdpr_preface', '1', { expires: Infinity });; location.reload(); }; });
```

Comment:
Uncaught TypeError: Cannot set property 'isvisible' of null

Better use this:
`warsteiner-gruppe.de#%#AG_onLoad(function() { var el = document.querySelector(".gdpr-preface"); if(el) { Cookies.set('gdpr_preface', '1', { expires: Infinity });; location.reload(); }; });`

Reply:
@BlazDT i've changed the rule, thanks